tests/test_pulsar.py

```diff
@@ -25,7 +25,7 @@
 
 
 @pytest.mark.skipif(
-    sys.platform in ("win32", "linux"),
+    sys.platform in ("win32",),
     reason="TODO Pulsar not setup properly on Windows/Linux yet",
```

Comment:
@yongtang The reason be changed only to `win32` here. Also, can we try this fix on win as well?

Reply:
It's because Pulsar server cannot run on Windows platform yet, see http://pulsar.apache.org/docs/en/standalone/#system-requirements

Reply:
I see. Thanks!